### ISSUE TITLE
How to refer to parent context when changing value?

### ISSUE THREAD

**Issue by oscarjlep** [sagalbot/vue-select#520]
I have looped rows with each row containing one v-select instance. Each v-select instance uses ajax-call for fetching array of items. I need to write a :on-change -callback, which would modify this parent-row-objects properties based on selected item. 

What would be the most sensible way to archive this? I'v been unable to pass parent as secondary object (something like :on-change="setItem(this, row)" and I haven't figured out how I could monitor which row is edited by using watchers.

```
<div v-for="row in rows">
    <v-select v-model="options.selectedItem" :placeholder="row.itemId" label="itemId" :on-change="setItem" @search="searchItemId" :options="options.items">

	<template slot="option" slot-scope="option">
		<span>{{ option.itemId }}</span><span>{{ option.itemName }}</span><span>{{ option.additionalItemName }}</span>
	</template>

    </v-select>
</div>
```

Basically setItem should update those three span-labels into parent row -objects fields by the same name (due to flattening).

**Comment by oscarjlep** [comment]
I resolved this by separating <v-select> to it's own sub-component (using it as an wrapper), to which I pass row as an property. This way each sub-component becomes it's own context and I'm able to target row-property from change-callback (or in my case by using watcher).
